vault/resource_okta_auth_backend.go

```diff
@@ -76,17 +80,23 @@ func oktaAuthBackendResource() *schema.Resource {
 			},
 
 			"ttl": {
-				Type:        schema.TypeString,
-				Required:    false,
-				Optional:    true,
-				Description: "Duration after which authentication will be expired",
+				Type:         schema.TypeString,
+				Computed:     true,
```

Comment:
So one behavior I'm noticing with these set to `Computed: true` is that if `ttl` or `max_ttl` is removed from the terraform manifest, those values are never unset in Vault. I'm not sure that's expected terraform behavior or not, so I'd be fine with setting a `Default: "0"` like you had it before instead.

Reply:
Interesting, I guess it makes sense for  `computed` values to never be persisted to the TF state. We can go with defaults for now, once we deprecate the `ttl` and `max_ttl` string inputs we should be in line with the other `auth_backend` resources that do not suffer from this issue.